creditcoin-js/src/utils.ts

```diff
@@ -53,3 +53,10 @@ export const utils = (api: ApiPromise) => {
         createOverrideWeight: () => createOverrideWeight(api),
     };
 };
+
+// block time in seconds
+export const BLOCK_TIME = 15;
```

Comment:
This is dependent on compiler flags. IMO best to request at runtime from the API. see 
https://github.com/gluwa/creditcoin/blob/dev/integration-tests/src/test/danger-will-brick-the-blockchain.test.ts#L34-L47